Games/core/Meeting.js

```diff
@@ -513,6 +529,29 @@ module.exports = class Meeting {
             this.game.spectatorsSeeVote(vote);
 
         this.checkReady();
+
+        if (this.game.vegKickMeeting === undefined || this.id === this.game.vegKickMeeting.id) {
+            return true;
+        }
+
+        // enough kicks reached
+        if (this.game.vegKickMeeting.finished) {
+            this.members[voter.id].canUpdateVote = false;
+            this.members[voter.id].canUnvote = false;
+            return true;
+        }
+
+        let player = this.members[voter.id].player;
+
+        if (this.game.vegKickMeeting.hasJoined(player)) {
+            return true;
+        }
+
+        if (player.hasVotedInAllMeetings()) {

```

Comment:
the first scenario is not a bug, because the sheriff was voting for a dead person in the village meeting.
tested that sheriff gets kicks after shooting if they voted for someone outside of their shot.